.github/workflows/deploy.yml

```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
 jobs:
   deploy-staging:
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
Will this also work when this workflow is run because something was pushed to the `main` branch? I suspect it won't. If it should, then something that hardcodes the official repo like https://github.com/web-platform-tests/wpt/pull/29195/files might be needed.

Reply:
Good catch!